apps/iiif/serializers/all_volumes_manifest.py

```diff
@@ -0,0 +1,32 @@
+# pylint: disable = attribute-defined-outside-init, too-few-public-methods
+"""Module for serializing all volumes manifest"""
+from django.core.serializers.base import SerializerDoesNotExist
+import config.settings.local as settings
+from apps.iiif.serializers.base import Serializer as JSONSerializer
+
+class Serializer(JSONSerializer):
+    """IIIF All volumes manifest"""
```

Comment:
I might change the docstring to something like `"""Serializer for a single IIIF manifest within the Readux all-volumes collection"""`